tests/integration/test_sqs.py

```diff
@@ -361,6 +361,33 @@ def test_batch_send_with_invalid_char_should_succeed(self, sqs_client, sqs_creat
         result_send = sqs_client.send_message_batch(QueueUrl=queue_url, Entries=batch)
         assert len(result_send["Failed"]) == 1
 
+    @pytest.mark.skipif(
```

Comment:
nit: We could use the `@only_localstack` decorator here (from `fixtures.py`).

Reply:
yes we should definitely use that :)